citesphere/src/main/webapp/WEB-INF/views/auth/group/items.jsp

```diff
@@ -146,7 +146,18 @@ $(function() {
 </div>
 
 <div class="col-md-2">
-<p class="lead">Collections</p>
+<p class="lead">Collections &nbsp;&nbsp;
+<c:choose>
+    <c:when test="${collectionId!=null}">
+        <a href="<c:url value="/auth/group/${zoteroGroupId}/collection/${collectionId}/items/sync?page=${currentPage}&sort=${sort}&columns=${columns}" />"><i
```

Comment:
columns need to be iterated over and not printed with brackets, this will solve the uglyness with the columns in your controller